http-driver/src/main/java/org/neo4j/ogm/drivers/http/request/HttpRequest.java

```diff
@@ -293,6 +295,8 @@ private static String parseError(String results) {
             } else {
                 return results;
             }
+        } catch (JsonParseException e) {
+            return "Unknown error";
```

Comment:
Struggling with me to decide if just a string with this content is enough detail.